mappings/net/minecraft/world/dimension/PortalForcer.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_467 net/minecraft/world/dimension/PortalForcer
+	FIELD field_1772 random Ljava/util/Random;
+	METHOD method_1530 force (Lnet/minecraft/class_18;Lnet/minecraft/class_57;)V
```

Comment:
maybe `changeDimension` or `teleport` would sound a bit nicer ? Altho once again its Notch name. Just a suggestion however. Approving anyway

Reply:
changeDimension doesn't make sense, because it doesn't change dimension, and teleport is half the story considering its teleporting to a portal pos, and I just found teleportToPortalPos a little annoying, so I just named it the action that this class is doing and gave it a comment "Called when an entity's dimension changes so that their position is forced to a portal position." especially since this method cannot fail because the entity's dimension has already changed.

Reply:
oki doki, I approved anyway, just planted a little mind bug while doing so.

Reply:
done